py/server/deephaven/table.py

```diff
@@ -1455,6 +1455,10 @@ def key_columns(self) -> List[str]:
             self._key_columns = list(self.j_partitioned_table.keyColumnNames().toArray())
         return self._key_columns
 
+    def keys(self) -> Table:
+        """Returns a Table containing all the keys of the underlying partitioned table."""
+        return self.table.select_distinct(self.key_columns)
```

Comment:
If `uniqueKeys` this should just be `view` (or maybe `select`), not `select_distinct`.